zookeeper-contrib/zookeeper-contrib-zooinspector/src/main/java/org/apache/zookeeper/inspector/manager/ZooInspectorManagerImpl.java

```diff
@@ -872,4 +881,26 @@ public Properties getLastConnectionProps() {
     public void setLastConnectionProps(Properties connectionProps) {
         this.lastConnectionProps = connectionProps;
     }
+
+    private static BufferedReader GetReaderForFile(File file) {
```

Comment:
please use Java convention `getReaderForFile`

Reply:
Thanks for catching that.  My apologies.